src/qml/pages/node/NodeSettings.qml

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import "../../controls"
+import "../../components"
+import "../settings"
+
+Page {
+    id: nodeSettings
+    property alias navMiddleDetail: node_settings.navMiddleDetail
+    property alias navRightDetail: node_settings.navRightDetail
+    background: null
+    StackView {
+        id: nodeSettingsView
+        initialItem: node_settings
+        anchors.fill: parent
+    }
+    Page {
+        id: node_settings
+        property alias navMiddleDetail: navbar.middleDetail
+        property alias navRightDetail: navbar.rightDetail
+        background: null
+        header: NavigationBar {
+            id: navbar
+        }
+        ColumnLayout {
+            spacing: 0
+            width: parent.width
+            ColumnLayout {
```

Comment:
Seems like one of these ColumnLayouts can be removed

Reply:
this allows for the table to expand but be a max width of 450, we use the same nested approach in the `InformationPage` control; open to better ways of accomplishing this

Reply:
Can use width: Math.min(parent.width, 450)